graph_generation/graph_template.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
Is this factory really required?

Reply:
It was more useful with more graph types, but if we will leave just proto and networkx it makes sense to remove it.